src/sentry/integrations/aws_lambda/integration.py

```diff
@@ -250,16 +250,15 @@ def dispatch(self, request: Request, pipeline) -> Response:
             return pipeline.next_step()
 
         organization = pipeline.organization
```

Comment:
Hold up -- `RpcOrganization` already contains projects as sub attributes, do we possibly already have this information here in the `pipeline.organization`?